packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -159,11 +159,11 @@
 			:downstream-operators-nav="downstreamOperatorsNav"
 			:spawn-animation="drilldownSpawnAnimation"
 			@append-output="(event: any) => appendOutput(currentActiveNode, event)"
+			@close="addOperatorToRoute(null)"
+			@select-output="(event: any) => selectOutput(currentActiveNode, event)"
+			@update-output="(event: any) => updateOutput(currentActiveNode, event)"
 			@update-state="(event: any) => updateWorkflowNodeState(currentActiveNode, event)"
 			@update-status="(status: OperatorStatus) => updateWorkflowNodeStatus(currentActiveNode, status)"
-			@select-output="(event: any) => selectOutput(currentActiveNode, event)"
-			@close="addOperatorToRoute(null)"
-			@update-output-port="(event: any) => updateOutputPort(currentActiveNode, event)"
```

Comment:
@YohannParis what is the rationale behind these name changes? 

There are operator components that will emit `update-output-port` , see https://github.com/search?q=repo%3ADARPA-ASKEM%2Fterarium%20update-output-port&type=code so they are going to break.


I think we should revert here and focus on the save-as feature, and not try to to do too many different things all at once.

Reply:
As you wish, I will make a small PR just for that renaming.